docs/manual/introduction/Optimizations.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<base href="../../" />
+		<script src="list.js"></script>
+		<script src="page.js"></script>
+		<link type="text/css" rel="stylesheet" href="page.css" />
+	</head>
+	<body>
+		<h1>[name]</h1>
+
+		<p class="desc">
+			This article shows how to optimize your three.js scene.
+		</p>
+
+		<h2>Metrics</h2>
+
+		<p>
+			Before starting, we need to define a some metrics, and how to monitor them. There are two important factors
+			that will affect your performances:
+
+			<ul>
+				<li>CPU: affects the number of frames per seconds (FPS) which represents how fluid is your scene</li>
```

Comment:
Why do you assign the performance metric FPS to CPU?

Reply:
Maybe I am wrong but IMO user is wondering about the FPS which is affected by CPU, for example, your frame 100ms to render.

Reply:
Everything affects the FPS. CPU and GPU, Memory bandwidth and compute power.

- CPU - lots of computation, like figuring out physics for particles
- main memory - your garbage collection can cause a stutter
- GPU computation - shaders can be heavy
- GPU memory - broad topic